pages/events/[event].tsx

```diff
@@ -121,44 +133,84 @@ const EventPage: NextPage = () => {
         // This redirect won't work on localhost (can't add dynamic URLs to
         // Supabase's allow-list), but does work in prod
         redirect={
-          eventID
-            ? `${process.env.NEXT_PUBLIC_HOSTNAME || ""}/events/${eventID}`
-            : undefined
+          eventID ? `${process.env.NEXT_PUBLIC_HOSTNAME || ""}/events/${eventID}` : undefined
         }
       />
     );
   }
-
+  let startDateStr = "",
+    endDateStr = "";
+  let startDate = new Date();
+  if (eventData) {
```

Comment:
If `!eventData && dateStatus !== DateStatus.loading`, I think we'd display some incorrect stuff below